### PR TITLE
Feature/edu 20 reset db migrations

### PR DESCRIPTION
**Pull request by ahelmberger**

### CODE REVIEW

File: src/bootstrap/server-config.js [hunk 1]
```diff
@@ -1,75 +1,56 @@
-/* eslint no-process-env: off */
-import parseBool from 'parseboolean';
-import Logger from '../common/logger.js';
+import joi from 'joi';
+import uniqueId from '../utils/unique-id.js';
+import { validate } from '../domain/validation.js';
 
-const logger = new Logger(import.meta.url);
+const configSchema = joi.object({
```
Comment (adrianaluchian) [inline src/bootstrap/server-config.js:5]: Nice!

File: migrations/educandu-2021-11-05-01-initial-schema.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
+// eslint-disable-next-line camelcase
+export default class Educandu_2021_11_05_01_initial_schema {
```
Comment (laurentiu-ilici) [inline migrations/educandu-2021-11-05-01-initial-schema.js:2]: New naming schema? I like it :)

File: migrations/educandu-2021-11-05-01-initial-schema.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        name: '_idx_username_provider_',
+        key: { username: 1, provider: 1 },
+        unique: true
+      },
+      {
+        name: '_idx_email_provider_',
+        key: { email: 1, provider: 1 },
+        unique: true
+      },
+      {
```
Comment (laurentiu-ilici) [inline migrations/educandu-2021-11-05-01-initial-schema.js:68]: Is this index really needed?

**Review by laurentiu-ilici — APPROVED** [review]

File: migrations/educandu-2021-11-05-01-initial-schema.js (hunk 3 above)
Comment (ahelmberger) [reply migrations/educandu-2021-11-05-01-initial-schema.js:68]: Because of the auto-expiry we need the index

**Pull request merged**